.betterer.ts

```diff
@@ -3,18 +3,14 @@ import { eslint } from '@betterer/eslint';
 
 export default {
   'no enzyme tests': () => regexp(/from 'enzyme'/g).include('**/*.test.*'),
-  'no explicit any': () =>
+  'better typescript': () =>
```

Comment:
two minds about naming it this (esp. given the churn on the graphite stats), but i think this name shouldn't be typescript-specific. `better eslint`? 😬